<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GEMI</class>
 <widget class="QDialog" name="GEMI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1066</width>
    <height>596</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GEMI</string>
  </property>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>110</y>
     <width>1021</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 0)

</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="BtnObjectifs">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>110</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>Objectifs</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BtnProgramme">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>110</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Programme</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BtnCooridnateur">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>110</y>
     <width>251</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Coordinateur</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BtnCompetences">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>110</y>
     <width>251</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Competences visées </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>10</y>
     <width>441</width>
     <height>71</height>
    </rect>
   </property>
   <property name="whatsThis">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:12pt; font-weight:700; font-style:italic;&quot;&gt;Cycle&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 28pt &quot;Rage Italic&quot;;
font: 700 36pt &quot;Script MT Bold&quot;;</string>
   </property>
   <property name="text">
    <string>Fillière : GEMI</string>
   </property>
  </widget>
  <widget class="QLabel" name="contenuObj">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>200</y>
     <width>981</width>
     <height>171</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size:14px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="labelCoordi">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>150</y>
     <width>961</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="labelCompetences">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>1121</width>
     <height>341</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size:14px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>110</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/assets/responsable-des-ressources-humaines.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>8</x>
     <y>115</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/assets/objectif.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>258</x>
     <y>115</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/assets/temps-detude.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>518</x>
     <y>115</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/assets/capacites.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>BtnObjectifs</tabstop>
  <tabstop>BtnProgramme</tabstop>
  <tabstop>BtnCompetences</tabstop>
  <tabstop>BtnCooridnateur</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
